packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -701,4 +702,16 @@ def self.add_flag_to_map_with_inheritance(map, field, flag)
             end
         end
     end
+
+    def self.resolve_use_frameworks(spec, header_mappings_dir: nil, module_name: nil)
+        if ENV['USE_FRAMEWORKS']
```

Comment:
nit:
I'd rather return early to have a less indented code after this.
```suggestion
        return unless ENV['USE_FRAMEWORKS'] 
```